sdk/freestanding-execution-engine/src/main.rs

```diff
@@ -254,5 +272,29 @@ fn main() -> Result<(), Box<dyn Error>> {
     let return_code = execute(&cmdline.execution_strategy, vfs.clone(), &prog_file_name)?;
     info!("return code: {:?}", return_code);
     info!("time: {} micro seconds", main_time.elapsed().as_micros());
+
+    // Dump contents of stdout
+    let buf = vfs.lock()
```

Comment:
Can we have a command-line option to enable this dumping of `stdout` and `stderr`?